src/oop/special_class_method/medium/q13/QuestionText.md

```diff
@@ -0,0 +1,26 @@
+# oop-special_class_method-medium-q13
+
+## Question Text
+
+You are developing a wholesale application consisting of items that can be purchased in large quantities. Make a Java 
+class called **Detergent** that has two attributes called **colour** and **volume**
```

Comment:
The attributes here do not match everywhere else. However, I do think "volume" is more relevant than "colour" in which case the attributes should probably become "volume" and "manufacturer"